conf/kamailio/kamailio.cfg

```diff
@@ -987,35 +987,3 @@ onreply_route[MANAGE_REPLY] {
     }
 }
 
-# Manage failure routing cases
-failure_route[MANAGE_FAILURE] {
-    route(NATMANAGE);
-
-    if (t_is_canceled()) exit;
-
-#!ifdef WITH_BLOCK3XX
-    # block call redirect based on 3xx replies.
-    if (t_check_status("3[0-9][0-9]")) {
-        t_reply("404","Not found");
-        exit;
-    }
-#!endif
-
-#!ifdef WITH_BLOCK401407
-    # block call redirect based on 401, 407 replies.
-    if (t_check_status("401|407")) {
-        t_reply("404","Not found");
-        exit;
-    }
-#!endif
-
-#!ifdef WITH_VOICEMAIL
-    # serial forking
-    # - route to voicemail on busy or no answer (timeout)
-    if (t_check_status("486|408")) {
-        $du = $null;
-        route(TOVOICEMAIL);
-        exit;
-    }
-#!endif
```

Comment:
Your billing-specific implementation "looses" these functionalities, which are part of the default kamailio config file. Doesn't it make sense to keep the failure_route in the main kamailio config file, add a RATING_MANAGE_FAILURE route in the billing config file and route there from the kamailio one if none of the above routes resolve?

Reply:
![image](https://user-images.githubusercontent.com/1306974/77856914-b9baa100-71fa-11ea-8406-f58b99dfd7e9.png)
In that case I have the kamailio box (192.168.128.10) responding with 408 Req Timeout.
Any idea why is that?

Reply:
Do you have any of those `WITH_*` statement in the configuration file?

Reply:
None of them

Reply:
Then I don't get why you get `408`, because it is just a matter of reordering the routes and instructions, isn't it?

Reply:
@tranchitella seems like a local issue on my computer. Let's see if the tests pass on gitlab.

Reply:
Found the issue. Fixed it and cleaned up the kamailio conf using `WITH_CANYAN_RATING` to define the blocks used in the main kamailio.cfg.